src/PeoplePage.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+
+import { Person } from './types/Person';
+import { getPeople } from './api';
+import { Loader } from './components/Loader';
+import { NoPeople } from './NoPeople';
+import { LoadingError } from './components/LoadError';
+import { TablePeople } from './components/TablePeople';
+
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [newError, setNewError] = useState(false);
+  const [load, setLoad] = useState(true);
+
+  useEffect(() => {
+    getPeople()
+      .then((peopleFromServer) => {
+        setLoad(true);
```

Comment:
The 'setLoad(true)' inside the '.then' block is not necessary since loading is already set to true before the API call and should only be set to false after the data is received or there's an error.